src/proof/drat/drat_proof.h

```diff
@@ -77,11 +77,29 @@ class DratProof
   ~DratProof() = default;
 
   /**
-   * Parses a DRAT proof from the binary format.
+   * Parses a DRAT proof from the **binary format**.
    * The format is described at:
    *    https://www.cs.utexas.edu/~marijn/drat-trim/#contact
    *
-   * @param binaryProof a string containing the bytes of the "binary" proof
+   * What do the standard authors mean by the format being "binary"?
+   * They just mean that proofs in this format should be understood as
+   * sequences of bytes, not sequences of ASCII/Unicode/your favorite character
+   * set characters.
+   *
+   * @param binaryProof a string containing the bytes of the binary proof.
+   *        Even though the proof isn't text, its safe to store it in a string
+   *        because C++ strings don't make any gaurantees about the encoding of
+   *        their contents. This makes them (effectively) just byte sequences.
+   *
+   *        A brief aside: In languages where strings are semantically
```

Comment:
I would keep the side note out of the source files (but maybe leave it in the discussion of this PR).

Reply:
Sounds good. I'll put the discussion in a comment.